conf/messages

```diff
@@ -464,6 +464,9 @@ label.no_record_of_employment=HMRC has no record of you being employed in this t
 label.not_enrolled.content=You are registered for Self Assessment. To use your Self Assessment online, you need to request access.
 label.not_enrolled.link.text=Request access to your Self Assessment
 
+title.seiss = Self-Employment Income Support Scheme (SEISS)
+body.seiss = View your SEISS grants (opens in new tab)
```

Comment:
Are there supposed to be spaces around the ' = '? Other messages don't have any spaces, so not sure if this affects the kerning